src/screens/Blocked/index.js

```diff
@@ -69,32 +70,34 @@ const Blocked = ({navigation}) => {
   }, []);
 
   return (
-    <View style={styles.containerTab}>
-      {headerBlocked()}
-      <Tabs.Navigator initialRouteName={TAB_BLOCKED_USER} tabBar={myTabbar}>
-        <Tabs.Screen
-          name={TAB_BLOCKED_USER}
-          component={BlockedUserList}
-          options={{
-            title: 'User'
-          }}
-        />
-        <Tabs.Screen
-          name={TAB_BLOCKED_DOMAIN}
-          component={BlockedDomainList}
-          options={{
-            title: 'Domain'
-          }}
-        />
-        {/* <Tabs.Screen
+    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.white}}>
+      <View style={styles.containerTab}>
+        {headerBlocked()}
+        <Tabs.Navigator initialRouteName={TAB_BLOCKED_USER} tabBar={myTabbar}>
+          <Tabs.Screen
+            name={TAB_BLOCKED_USER}
+            component={BlockedUserList}
+            options={{
+              title: 'User'
+            }}
+          />
+          <Tabs.Screen
+            name={TAB_BLOCKED_DOMAIN}
+            component={BlockedDomainList}
+            options={{
+              title: 'Domain'
+            }}
+          />
+          {/* <Tabs.Screen
             name={TAB_BLOCKED_TOPIC}
             component={BlockedTopicList}
             options={{
                 title: 'Topic',
               }}
             /> */}
-      </Tabs.Navigator>
-    </View>
+        </Tabs.Navigator>
+      </View>
+    </SafeAreaView>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `SafeAreaView` is a good practice for handling view areas that might be obstructed by system UI, notches, or sensor clusters. However, the inline style used here could be moved to the StyleSheet object for better organization and reusability.
```diff
- <SafeAreaView style={{flex: 1, backgroundColor: COLORS.white}}>
+ <SafeAreaView style={styles.safeArea}>

// Add this to your styles object
const styles = StyleSheet.create({
   safeArea: {
     flex: 1,
     backgroundColor: COLORS.white
   },
   // ... other styles
});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->